Cargo.toml

```diff
@@ -32,19 +32,14 @@ edition = "2021"
 description = "Top level crate for Blitz"
 license = "MIT OR Apache-2.0"
 keywords = ["dom", "ui", "gui", "react", "wasm"]
-rust-version = "1.60.0"
+rust-version = "1.70.0"
 publish = false
 
 [profile.dev.package."*"]
 opt-level = 2
 
 # Need to force specific versions of these dependencies
 [dependencies]
-# webrender = "0.61.0"
-euclid = { version = "0.22", features = ["serde"] }
-# mozbuild = "0.1.0"
-
-[dev-dependencies]
```

Comment:
What's the reason for moving all of these deps from "dev-dependencies" to "dependencies"?